mode/threads.py

```diff
@@ -179,13 +181,12 @@ async def start(self) -> None:
 
     async def _keepalive2(self) -> None:
         while not self.should_stop:
-            await self.sleep(1.1)
+            await self.sleep(2.0)
```

Comment:
Wish I could understand why `1.1` and `2.0` are magical values that make everything work.

Reply:
No, simply nothing, just the interval of wake-up. Actually all number smaller than `3.0` int the following `monotonic() - self.last_wakeup_at > 3.0` is fine. I try to increase to `2.0` to reduce the frequency.

Reply:
Yeah, you can add some notes or comments here.

Reply:
Didn't see this until now, https://github.com/faust-streaming/mode/blob/17c5d2996e4706baa37b18bd8fa03687409120ad/mode/timers.py#L13
Looks like we'll need to go back to `1.1`.

Reply:
OK